### PR TITLE
Clean-up frame buffer definition

### PR DESCRIPTION
**Pull request by palemieux**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6920,7 +6920,7 @@ <h2>Decoder gamma
 calibration measurements are available for the particular <a>CRT</a>
 used.</p>
 
-<p>Many modern frame buffers have lookup tables that are used to
+<p>Many modern <a>frame buffer</a>s have lookup tables that are used to
```
Comment (ProgramMax) [inline index.html:6923]: I think ReSpec is okay with plurals. If you haven't tried, can you see if including the 's' is fine?

**Review by ProgramMax — CHANGES_REQUESTED** [review]
LGTM, but try a change for me

File: index.html (hunk 1 above)
Comment (palemieux) [reply index.html:6923]: Thanks for the catch!

Comment (palemieux) [reply index.html:6923]:
```suggestion
<p>Many modern <a>frame buffers</a> have lookup tables that are used to
```

**Review by ProgramMax — APPROVED** [review]
LGTM

**Pull request merged**